sharding/handler/log_handler.py

```diff
@@ -1,55 +1,44 @@
 import logging
 
-from sharding.handler.utils.web3_utils import (
-    get_recent_block_hashes,
-    get_canonical_chain,
-)
-
 
 class LogHandler:
 
-    logger = logging.getLogger("evm.chain.sharding.LogHandler")
+    logger = logging.getLogger("sharding.handler.LogHandler")
 
-    def __init__(self, w3, history_size=256):
-        self.history_size = history_size
+    def __init__(self, w3, period_length):
         self.w3 = w3
-        # ----------> higher score
-        self.recent_block_hashes = get_recent_block_hashes(w3, history_size)
-
-    def get_new_logs(self, address=None, topics=None):
-        # TODO: should see if we need to do something with revoked_hashes
-        #       it seems reasonable to revoke logs in the blocks with hashes in `revoked_hashes`
-        revoked_hashes, new_block_hashes = get_canonical_chain(
-            self.w3,
-            self.recent_block_hashes,
-            self.history_size,
-        )
-        # determine `unchanged_block_hashes` by revoked_hashes
-        # Note: use if/else to avoid self.recent_block_hashes[:-1 * 0]
-        #       when len(revoked_hashes) == 0
-        if len(revoked_hashes) != 0:
-            unchanged_block_hashes = self.recent_block_hashes[:-1 * len(revoked_hashes)]
+        self.period_length = period_length
+
+    def get_logs(
+            self,
+            *,
```

Comment:
Nice to see this cool usage of asterisk! But I don't see why we need it here?

Reply:
Not really. Just personal preference toward passing parameter along with keyword.